Kudu.Core/Deployment/DeploymentManager.cs

```diff
@@ -544,6 +544,27 @@ private DeployResult GetResult(string id, string activeDeploymentId, bool isDepl
                 string repositoryRoot = _environment.RepositoryPath;
                 var perDeploymentSettings = DeploymentSettingsManager.BuildPerDeploymentSettingsManager(repositoryRoot, _settings);
 
+                string delayMaxInStr = perDeploymentSettings.GetValue(SettingsKeys.MaxRandomDelayInSec, false);
+                if (delayMaxInStr != null)
```

Comment:
we tend to use `string.IsNullOrEmpty(delayMaxInStr)` in kudu for string null checks. I know this is semantically different from `== null` but it's almost always what you want anyway.